packages/react-dom/src/__tests__/ReactServerRendering-test.js

```diff
@@ -795,37 +795,6 @@ describe('ReactDOMServer', () => {
     }
   });
 
-  it('warns about lowercase html but not in svg tags', () => {
-    function CompositeG(props) {
-      // Make sure namespace passes through composites
-      return <g>{props.children}</g>;
-    }
-    expect(() =>
-      ReactDOMServer.renderToStaticMarkup(
-        <div>
-          <inPUT />
-          <svg>
-            <CompositeG>
-              <linearGradient />
-              <foreignObject>
-                {/* back to HTML */}
-                <iFrame />
-              </foreignObject>
-            </CompositeG>
-          </svg>
-        </div>,
-      ),
-    ).toErrorDev([
```

Comment:
Can you leave the test be but without `toErrorDev`? So that we track we do _not_ warn for `linearGradient` and friends still. In case warning gets re-added.